### DISCUSSION TITLE
Is this still in development?

### DISCUSSION THREAD

**Discussion by apexiptv** [oddbear/GoXLR-TouchPortal-Plugin#32] [General]
There has been several updates on the goxlr software and firmware since December. Is this a dead project or is still a viable option for us touch portal users?

**Comment by oddbear** [comment]
Hi.

It should not be any changes to the API, latest version (v0.18 released 27/10/2021) is the same version used in the current beta (released 16/08/2022).

So there should not be any new updates, and no issues has been reported.

If you see any updates or issues, please keep me posted if I miss it. :)

If you want any new features, the best place to ask for them is in the feature request channel on the TC-Helicon discord, as this plugin is (mostly) limited to the same feature set as the official Stream Deck plugin.

**Reply by apexiptv** [reply]
I would love to see slidier volume options via the TP plugin... As Voicemeter and other plugins have enabled...

**Reply by oddbear** [reply]
That would be awesome, but the Stream Deck API is to limited at this time. I hope this could be added in the future.

**Reply by oddbear** [reply]
I could add to this that I have also been working on plugins for the PreSonus Revelator series and the Elgato Wave XLR. Having features like you mentioned there, makes it really hard to switch back to the GoXLR as my main interface. Even though both of these interfaces has other issues.
